doc/liblouis.texi

```diff
@@ -977,17 +923,8 @@ require special treatment.
 
 @table @code
 @opcode{decpoint, character dots}
-This opcode defines the decimal point. It is useful if your Braille
```

Comment:
Why did you remove this section? It contains some additional explanation about the decpoint. Is it wrong?